mathics/builtin/specialfunctions.py

```diff
@@ -858,3 +858,34 @@ def prepare_sympy(self, leaves):
 #    nargs = 3
 #    sympy_name = ''
 #    mpmath_name = ''
+
+
+class ExpIntegralEi(_MPMathFunction):
+    '''
+    <dl>
+    <dt>'ExpIntegralEi[$z$]'
+      <dd>returns the exponential integral function $Ei(z)$.
+    </dl>
+
+    >> ExpIntegralEi[2.0]
+     = 4.95423435600189016
+    '''
+
+    sympy_name = ''
```

Comment:
I've opened https://github.com/sympy/sympy/pull/11341 to add this functionality to sympy.